test/e2e/bound_sa_token.go

```diff
@@ -0,0 +1,75 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+
+	g "github.com/onsi/ginkgo/v2"
+	"github.com/stretchr/testify/require"
+
+	authenticationv1 "k8s.io/api/authentication/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	testlibrary "github.com/openshift/cluster-kube-apiserver-operator/test/library"
+)
+
+var _ = g.Describe("[sig-api-machinery] kube-apiserver operator", func() {
+	g.It("[Operator] TestTokenRequestAndReview", func() {
+		TestTokenRequestAndReview(g.GinkgoTB())
+	})
+})
+
+// TestTokenRequestAndReview checks that bound sa tokens are correctly
+// configured. A token is requested via the TokenRequest API and
+// validated via the TokenReview API.
+func TestTokenRequestAndReview(t testing.TB) {
```

Comment:
let's make this method private `testTokenRequestAndReview(t testing.TB)`

Reply:
Updated